feat(ups grafana dash) : Upgrade it with the latest version

camilamacedo86 opened the pull request:
Ref: https://github.com/integr8ly/installation/pull/808

wei-lee commented on roles/ups/templates/grafana_dashboard.yml.j2:
```diff
@@ -6,6 +6,9 @@ metadata:
 spec:
+  selector:
+    matchLabels:
+      name: {{ ups_server_name }}
```
I am pretty sure this part is not required.

camilamacedo86 replied:
Oks we can remove it so.

wei-lee commented:
@camilamacedo86 how did you create the dashboard? Did you deploy it anywhere? This PR is hard to review at the moment. Also the JIRA wasn't really brought in for the sprint. Do we really need this version of dashboard? What things are missing/broken in the current one?

camilamacedo86 commented:
HI @wei-lee, 

Note that it was created 22 days ago here in the installer and the latest changes performed in the UPS on it was 15 days ago.  So, it cannot be the same content.

Why do we need to update it?
- Will be very hard applying the next requests/fixes and changes if it is not the same version
- I believe that the version of 15 days ago has bug fixes and addressing the comments made by you folks in the review. So, is really valid we spend time comparing the both? Is not better we merge it and then check the result? Because as you said will hard validate it another way now. I think that it was added here before we finish the tasks regards the dashboards there.

wei-lee commented:
@camilamacedo86 but do you know exactly what this PR fixes? Or can I see it somewhere. Just because the version is 15 days old doesn't mean there are problems with it.

camilamacedo86 commented:
Hi @wei-lee,

The changes was made in the PR: https://github.com/aerogear/unifiedpush-operator/commit/bd59a360a700a3d2cde9eb5c1b1f4daf9dabcac2#diff-31f10ce7bb67936fff50032371bb0bec. Note that here we addressed all comments, suggestions and requested made regards to it in the review of this graph. 

We changed many things will be hard we check all in the same way that will be hard we apply the other task do replace the % for an upside/down as any other one without keeping them both using the same versions.

wei-lee commented on roles/ups/templates/grafana_dashboard.yml.j2:
```diff
@@ -36,15 +36,15 @@ spec:
+              "iconColor": "rgba(0, 211, 255, 1)",
+              "name": "Annotations & Alerts",
+              "type": "dashboard"
+            }
```
datasource is not required. Let's not add them.

wei-lee replied:
nvm. Wrong section.

camilamacedo86 replied:
👍

wei-lee commented:
@camilamacedo86 ok. can you address my last comment and then merge this PR. We can see how it works after provisioned to a cluster.
